__tests__/app.test.js

```diff
@@ -190,3 +190,34 @@ describe("/api/articles/:article_id/comments", () => {
       .expect(400);
   });
 });
+describe("PATCH /api/articles/:article_id", () => {
+  test("PATCH:200 - increment the votes by votes_count", () => {
+    return request(app)
+      .patch("/api/articles/2")
+      .send({ inc_votes: 10 })
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article).toMatchObject({
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+        expect(article.votes).toEqual(10);
+      });
```

Comment:
This is a valid test, however, you can also hard code values in `toMatchObject` So votes can be 10 there instead of `expect.any(Number)` Then you don't need the extra assertion you have to check the article.votes. It would also be nice to see the article if in the object and check that is indeed 2